src/Dockerfile

```
@@ -0,0 +1,16 @@
+FROM alpine:latest
+
+LABEL maintainer="Antonio Tauro <toni.tauro@adfinis.com>"
```

Please use [opencontainers labels](https://github.com/opencontainers/image-spec/blob/master/annotations.md#pre-defined-annotation-keys):
```suggestion
LABEL org.opencontainers.image.authors="Antonio Tauro <toni.tauro@adfinis.com>"
```